src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -511,6 +512,10 @@
 						eventUtils.trigger(state.items[state.currentIndex].element, "selected");
 						state.toIndex = state.currentIndex;
 
+						// set last scroll position when current page is hidden
+						pageWidget = ns.engine.getBinding(self._ui.page, "Page");
+						pageWidget.setLastScrollPosition(-1 * scroll.current || 0);
```

Comment:
Who sets _lastScrollPosition to 0 when page is hide?
Shouldn't we sync page's scroller.scrollTop with ArcListView's scroll.current instead?

Reply:
The _lastScrollPosition is set by Page widget on "pagebeforehide" and restored on "pagebeforeshow". The ArcListview try to change scroll position but couldn't. Now, new position of last scroll position can be changed even for hidden page.